packages/flutter/lib/src/material/material.dart

```diff
@@ -153,6 +155,7 @@ class Material extends StatefulWidget {
     this.shadowColor: const Color(0xFF000000),
     this.textStyle,
     this.borderRadius,
+    this.shapeBorder,
```

Comment:
let's just call this "shape" for consistency with other places where we take ShapeBorder

Reply:
done, to be completely consistent I also renamed shapeBorder to shape in ShapeBorderClipper